### DISCUSSION TITLE
iCUE H60i RGB PRO XT support?

### DISCUSSION THREAD

**Discussion by kevitivity** [liquidctl/liquidctl#519] [General]
I'm guessing this particular AIO is not supported, but just wanted to verify here.

I'm running ubuntu 22. lsusb sees: H60iRGBPROXT

**Comment by jonasmalacofilho** [comment]
Yes, it should be, provided you're running a suitably recent version of liquidctl: 4ebc55b8e9ba8deebd5682f7f3b46e034f328a9b (#402).

**Comment by kevitivity** [comment]
ubuntu 22 provides liquidctl liquidctl v1.5.1. I will try compiling from source.

**Reply by jonasmalacofilho** [reply]
Sadly true.

Any chance you can file a "please update" report to Ubuntu?  (I think there's one already for Debian, but there hasn't been any activity...)

**Reply by kevitivity** [reply]
Yes, I will.  v1.10.0 worked perfectly. 

Thank you for this project!

**Comment by kevitivity** [comment]
TY for reply btw.
